java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -2119,11 +2119,11 @@ void testStringSplitRecord() {
       try (ColumnVector v = ColumnVector.fromStrings("Héllo there", "thésé", "null", "", "ARé some", "test strings");
            ColumnVector expected = ColumnVector.fromLists(
                    new HostColumnVector.ColumnBuilder.ListType(true, 6,
-                           new HostColumnVector.ColumnBuilder.BasicType(true, 8, DType.STRING)),
+                           new HostColumnVector.ColumnBuilder.BasicType(true, 9, DType.STRING)),
                    Arrays.asList("Héllo", "there"),
                    Arrays.asList("thésé"),
                    Arrays.asList("null"),
-                   Arrays.asList(),
+                   Arrays.asList(""),
                    Arrays.asList("ARé", "some"),
```

Comment:
These 2 line changes seem to be in line with cudf cpp tests like below where empty entries like "" are also part of the o/p.  @revans2 please correct me if I missed something here.

Reply:
```
TEST_F(StringsSplitTest, SplitRecord)
{
  std::vector<const char*> h_strings{" Héllo thesé", nullptr, "are some  ", "tést String", ""};
  auto validity =
    thrust::make_transform_iterator(h_strings.begin(), [](auto str) { return str != nullptr; });
  cudf::test::strings_column_wrapper strings(h_strings.begin(), h_strings.end(), validity);

  auto result =
    cudf::strings::split_record(cudf::strings_column_view(strings), cudf::string_scalar(" "));
  using LCW = cudf::test::lists_column_wrapper<cudf::string_view>;
  LCW expected(
    {LCW{"", "Héllo", "thesé"}, LCW{}, LCW{"are", "some", "", ""}, LCW{"tést", "String"}, LCW{""}},
    validity);
  cudf::test::expect_columns_equal(result->view(), expected);
}
```